neoexchange/neox/settings.py

```diff
@@ -7,7 +7,7 @@
 import rollbar
 
 
-VERSION = '3.9.1'
+VERSION = '3.10'
```

Comment:
Should be `3.10.0` to make `major.minor.revision` standard

Reply:
@talister I did this 2 days ago in "fix version number" commit 3c8dc86

Is it not showing up for you? :/

Reply:
It is now, must have clicked on the wrong part of the PR timeline or it's refreshed itself since